apps/back-office/src/app/(dashboard)/calendar/components/calendar/calendar.tsx

```
@@ -148,6 +161,7 @@ export function CalendarView({
       // For availability events, only check clinician
       if (event.extendedProps?.type === "availability") {
         return event.resourceId;
+        return event.resourceId;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)